lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -99,13 +118,36 @@ class tethys::ScienceSensorsSystemPrivate
   public: std::string dataPath {"2003080103_mb_l3_las.csv"};
 
   /// \brief Indicates whether data has been loaded
-  public: bool initialized = false;
+  public: bool initialized {false};
+
+  /// \brief Set to true after the spherical coordinates have been initialized.
+  /// This may happen at startup if the SDF file has them hardcoded, or at
+  /// runtime when the first vehicle is spawned. Assume the coordinates are
+  /// only shifted once.
+  public: bool worldSphericalCoordsInitialized {false};
+
+  /// \brief Mutex for writing to world origin association to lat/long
+  public: std::mutex mtx;
+
+  /// \brief Spherical coordinates of world origin. Can change at any time.
+  public: ignition::math::SphericalCoordinates worldOriginSphericalCoords;
+
+  /// \brief World origin in spherical position (latitude, longitude,
+  /// elevation), angles in degrees
+  public: ignition::math::Vector3d worldOriginSphericalPos = {0, 0, 0};
```

Comment:
Do we need to store this variable? I don't see it being used anywhere besides right after it is set.

Reply:
I print this out in `ReadData()` in #83. That might be another forward-merge thing. I think this is a useful variable that I'll be using for debugging until all the sensor stuff is finalized. We're still pretty far from that, at least 2 more PRs, before actually testing the missions.

Reply:
I think it's just holding duplicate data that's already in `worldOriginSphericalCoords`, right? Couldn't you just print the values from that? Keeping the same information in 2 places is prone to error because we need to keep them in sync.